iroh-relay/src/lib.rs

```diff
@@ -46,6 +46,7 @@ pub(crate) use key_cache::KeyCache;
 
 #[cfg(not(wasm_browser))]
 pub mod dns;
+pub mod node_info;
```

Comment:
Moving `node_info` from `crate::dns::node_info` to `crate::node_info`, because we need it for discovery in browsers, but it's still nice to disable the whole `dns` module in browsers, too.